src/librepgp/stream-write.cpp

```diff
@@ -1644,8 +1633,8 @@ init_literal_dst(pgp_write_handler_t *handler, pgp_dest_t *dst, pgp_dest_t *writ
 {
     pgp_dest_packet_param_t *param;
     rnp_result_t             ret = RNP_ERROR_GENERIC;
-    int                      flen = 0;
     uint8_t                  buf[4];
+    size_t                   flen = 0;
```

Comment:
why is this moved?

Reply:
@rrrooommmaaa Just typo - first attempted to move it below, where it is used for the first time, but then realized that there is a goto before. Should I revert this to the initial location?

Reply:
@ni4 would be much easier to track changes, evidently

Reply:
Okk, will re-push.